README.md

```diff
@@ -69,7 +69,8 @@ Extensions can override the following default extension points to add or change
     onEvent : function(name, evt) {return true;},
     transformResponse : function(text, xhr, elt) {return text;},
     isInlineSwap : function(swapStyle) {return false;},
-    handleSwap : function(swapStyle, target, fragment, settleInfo) {return false;},
+    handleSwap : function(swapStyle, target, fragment, settleInfo) {return [target];},
     encodeParameters : function(xhr, parameters, elt) {return null;}
+    getSelectors : function() {return [];}
```

Comment:
See the lib core for this as well:
https://github.com/bigskysoftware/htmx/blob/master/src/htmx.js#L4792

`getSelectors` returns `null` by default, I think we should reuse the same default values in this README

Reply:
Good Catch! I copied the `extensionBase` object over 😉

I also updated this in the type definition over at HTMX itself. See https://github.com/bigskysoftware/htmx/pull/2721#issuecomment-2226992513